packages/blade/src/components/DatePicker/CalendarStyles.web.tsx

```diff
@@ -0,0 +1,304 @@
+/* eslint-disable @typescript-eslint/restrict-plus-operands */
+import styled from 'styled-components';
+import dayjs from 'dayjs';
+import type { PickerType } from './types';
+import BaseBox from '~components/Box/BaseBox';
+import getTextStyles from '~components/Typography/Text/getTextStyles';
+import { size } from '~tokens/global';
+import { makeSpace } from '~utils';
+import getIn from '~utils/lodashButBetter/get';
+import { useIsMobile } from '~utils/useIsMobile';
+
+const cell = {
+  gap: {
+    x: size[0],
+    y: size[2],
+  },
+  size: {
+    desktop: size[40],
+    mobile: size[48],
+  },
+  background: {
+    default: 'transparent',
+    hover: 'interactive.background.gray.highlighted',
+    disabled: 'transparent',
+  },
+  text: {
+    default: 'interactive.text.gray.normal',
+    disabled: 'interactive.text.gray.disabled',
+  },
+} as const;
+
+const todayCell = {
+  background: {
+    default: 'transparent',
+    hover: 'interactive.background.primary.faded',
+  },
+  border: {
+    default: 'interactive.border.primary.default',
+    hover: 'interactive.border.primary.default',
+  },
+  text: {
+    default: 'interactive.text.primary.normal',
+    hover: 'interactive.text.primary.normal',
+  },
+} as const;
+
+const selectedCell = {
+  background: {
+    default: 'interactive.background.primary.default',
+    hover: 'interactive.background.primary.highlighted',
+  },
+  border: {
+    default: 'interactive.border.primary.default',
+    hover: 'surface.border.primary.muted',
+  },
+  text: {
+    default: 'interactive.text.onPrimary.normal',
+    hover: 'interactive.text.onPrimary.normal',
+  },
+} as const;
+
+const inRangeCell = {
+  background: {
+    default: 'surface.background.primary.subtle',
```

Comment:
this should be interactive no?

Reply:
will check

Reply:
There's no `interactive.surface.background.primary.subtle` this is going to be surface color only.